gcp-main (2).tf

```diff
@@ -0,0 +1,141 @@
+module "vpc" {
+    source = "./modules/gcp/vpc"
+    project_id = "playground-351903"
+    region = "us-east4"
+    network_name = "dev-microservices-new-vpc"
+    description = "custom vpc for kubernetes engine workloads"
+    subnets = [
+        {
+           subnet_name = "dev-microservices-new-public-subnet"
+           subnet_ip   = "10.10.0.0/16"
+           subnet_region = "us-east4"
+           subnet_private_access = false ##Setting to false as it should be act as public subnet
+           subnet_flow_logs      = "false"
```

Comment:
Is it necessary of this to be a string encoded boolean? Can we change this to bool values and do the encoding in the terraform module itself?